dashboard/views.py

```diff
@@ -28,14 +29,23 @@ def home(request):
     :return: The home page template or the login template.
     """
     if request.user.is_authenticated and is_integrated(request.user):
-        #  If not already present, store the experiment start date after the first login
+        #  Store the experiment start date after the first login and redirect to questionnaire
         if not StartDate.objects.filter(user=request.user).exists():
             StartDate(user=request.user).save()
+            # redirect to self-efficacy questionnaire
```

Comment:
This is the first entry point of the questionnaire.
We must hit it at the beginning of the experiment.